lib/modules/manager/npm/extract/index.ts

```diff
@@ -138,6 +143,19 @@ export async function extractPackageFile(
   const yarnrcYmlFileName = getSiblingFileName(fileName, '.yarnrc.yml');
   const yarnZeroInstall = await isZeroInstall(yarnrcYmlFileName);
 
+  let yarnConfig: YarnConfig | null;
```

Comment:
```suggestion
  let yarnConfig: YarnConfig | null = null;
```

otherwise it's undefined 😉